app/views/user_mailer/new_email.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to Teach for america tfa.org, <%= @user.name %></h1>
```

Comment:
Perhaps don't need a welcome if they're already signed up and just changing emails